cadc-vos/src/main/java/ca/nrc/cadc/vos/client/ClientTransfer.java

```diff
@@ -90,26 +88,18 @@
 import javax.net.ssl.SSLSocketFactory;
 import javax.security.auth.Subject;
 
+import ca.nrc.cadc.net.*;
```

Comment:
The convention in the opencadc code is to list each of the imports instead of using wildcard imports.